std/typecons.d

```diff
@@ -2185,6 +2185,69 @@ Params:
         _isNull = false;
     }
 
+    /**
+      If they are both null, then they are equal. If one is null and the other
+      is not, then they are not equal. If they are both non-null, then they are
+      equal if their values are equal.
+      */
+    bool opEquals()(auto ref inout(typeof(this)) rhs) inout
+    {
+        if(_isNull)
+            return rhs._isNull;
+        if(rhs._isNull)
+            return false;
+        return _value == rhs._value;
```

Comment:
Why would you do that? This is `opEquals`. It's _normal_ to access the member variables of the two variables to compare them. Calling `get` would just add unnecessary overhead.